### PR TITLE
add finish button to task detail view

### LINKED ISSUE #716: Task completion in task view
## What
Additionally to setting task to "done" via drop-down, a button or checkbox to finish a task (similar to subtasks) would be grand 

## Why
One more way to finish tasks and give users choice

### PR DESCRIPTION
**Pull request by DasProffi**
## Which issues does this pull request close?
closes #716

### CODE REVIEW

File: tasks/components/layout/TaskDetailView.tsx [hunk 1]
```diff
@@ -352,6 +355,15 @@ export const TaskDetailView = ({
...
                 </Button>
+                <Button
+                  color="positive"
+                  onClick={() => {
+                    toDoneMutation.mutate(task.id)
+                    onClose()
+                  }}
+                >
+                  {translation.finish}
+                </Button>
```
Comment (use-to) [inline tasks/components/layout/TaskDetailView.tsx:366]: Should not be clickable when task already in done

File: tasks/components/layout/TaskDetailView.tsx [hunk 2]
```diff
@@ -74,6 +75,7 @@ const defaultTaskDetailViewTranslation: Record<Languages, TaskDetailViewTranslat
     publish: 'Publish',
     publishTask: 'Publish Task',
     publishTaskDescription: 'This cannot be undone',
+    finish: 'Task finished',
```
Comment (use-to) [inline tasks/components/layout/TaskDetailView.tsx:78]:
```suggestion
    finish: 'Finish Task',
```

File: tasks/components/layout/TaskDetailView.tsx [hunk 3]
```diff
@@ -92,6 +94,7 @@ const defaultTaskDetailViewTranslation: Record<Languages, TaskDetailViewTranslat
     publish: 'Veröffentlichen',
     publishTask: 'Task Veröffentlichen',
     publishTaskDescription: 'Diese Handlung kann nicht rückgängig gemacht werden',
+    finish: 'Task abgeschlossen',
```
Comment (use-to) [inline tasks/components/layout/TaskDetailView.tsx:97]:
```suggestion
    finish: 'Fertigstellen',
```

**Review by use-to — CHANGES_REQUESTED** [review]

**Comment by florianjs** [comment]
Do we really need a locked button? We already have the dropdown, which allows the same thing

**Review by use-to — APPROVED** [review]

**Comment by florianjs** [comment]
Do we want to rethink the design with this butoon we would have 3 possibilities to set a task to Connected

**Pull request merged**